### ISSUE TITLE
UnicodeDecodeError

### ISSUE THREAD

**Issue by patois** [patois/abyss#8]
Traceback (most recent call last):
  File "G:/IDA75/plugins/abyss.py", line 152, in text_ready
    obj.process_text(vu)
  File "G:/IDA75/plugins\abyss_filters\ctopy.py", line 1331, in process_text
    lines = "\n".join([ida_lines.tag_remove(sl.line) for sl in pc])
  File "G:/IDA75/plugins\abyss_filters\ctopy.py", line 1331, in <listcomp>
    lines = "\n".join([ida_lines.tag_remove(sl.line) for sl in pc])
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xba in position 204: invalid start byte

_Originally posted by @x64bugreport in https://github.com/patois/abyss/issues/4#issuecomment-729387253_

**Comment by patois** [comment]
https://github.com/patois/abyss/issues/4#issuecomment-729527990

**Comment by patois** [comment]
@x64bugreport could you please attach a copy of the original pseudo-code before it is processed by the abyss plugin?

**Comment by x64bugreport** [comment]
![1111](https://user-images.githubusercontent.com/34920463/99527574-f7fa1000-29d7-11eb-977e-8e81f1acf054.png)
addr:0x401000
The problem may be caused by a pseudo-code page that contains comments.

[ConsoleApplication1.idb.zip](https://github.com/patois/abyss/files/5559796/ConsoleApplication1.idb.zip)

**Comment by x64bugreport** [comment]
I hope to fix the #4 bug soon. The problem test of #8 may be caused by special character comments.
If the problem of #8 cannot be avoided, I hope you try to use the try-except in the script to catch the problem and prompt the user.

**Comment by patois** [comment]
I am unable to reproduce this behavior since IDA refuses to open the IDB file you provided.

Could you run the following lines in IDA's Python CLI and let me know whether this reproduces the error?

```
from ida_lines import tag_remove
"\n".join([tag_remove(line) for line in ["syste\xbam", "test123\xba"]])
```

**Comment by x64bugreport** [comment]
hello,it is new idb.
[testidb.patched.zip](https://github.com/patois/abyss/files/5565030/testidb.patched.zip)

**Comment by patois** [comment]
I am able to reproduce the issue and it happens as soon as the "line" field of a simpleline_t instance is accessed. Although it should be possible to work around this issue using exception handling, this is not a desirable solution that I would implement into the plugin. The behavior is caused by and should be handled/fixed by @idapython instead.

**Comment by x64bugreport** [comment]
Thank you for your reply. It is recommended that you remind the user when the plugin is abnormal, instead of directly crashing. At the same time, I hope you can fix the problem #4. Thank you very much for writing the plugin, thank you!

**Comment by patois** [comment]
Thank you for having reporting this issue, I will close it for now.
